### PR TITLE
Add the landing page and sharing feature on profile view

### PR DESCRIPTION
**Pull request by YaiyaSam**
## Purpose

The purpose of this PR is to fix #<issue-number>

## Screenshots

image

### CODE REVIEW

File: src/scenes/PublicHome/index.tsx [hunk 1]
```diff
@@ -1,7 +1,16 @@
 function PublicHome() {
   return (
-    <div className="mx-auto md:mx-12">
-      Public Home
+    <div className="mx-auto md:mx-12 w-full h-full flex ">
+      <div className="w-1/3 h-full font-bold mt-30 ml-20">
+        <p className="text-blue-900 text-7xl mt-32">WeHuddle</p>
+        <p className="font-normal text-gray-500 text-lg mt-5">
+          lorem ipsum ah jlsao hdsal hdaskd lajds al dsk daskd askdjsad sadlsajd
+          sajdlsajd hkdsa daksjd kda sdk sadkjs djsakd akjsdkas jd
```
Comment (Gravewalker666) [inline src/scenes/PublicHome/index.tsx]: Let's put something meaningful here

**Review by Gravewalker666 — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -22,6 +22,7 @@
     "flowbite-react": "^0.1.3",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-helmet": "^6.1.0",
```
Comment (Gravewalker666) [inline package.json]: .

File: package.json [hunk 3]
```diff
@@ -53,6 +54,7 @@
     ]
   },
   "devDependencies": {
+    "@types/react-helmet": "^6.1.5",
```
Comment (Gravewalker666) [inline package.json]: What is this used for?

**Review by Gravewalker666 — CHANGES_REQUESTED** [review]
Some conflicts are there

File: package.json (hunk 3 above)
Comment (YaiyaSam) [reply package.json]: ah its something i used for the meta tags thing.

Comment (YaiyaSam) [reply package.json]: I removed it

**Review by Sanjana-99 — APPROVED** [review]

**Review by pasindur99 — APPROVED** [review]

**Pull request merged**